src/app/Shared/Redux/Configurations/DashboardConfigSlicer.tsx

```diff
@@ -74,8 +82,19 @@ export const dashboardCardConfigDeleteCardIntent = createAction(DashboardConfigA
   } as DashboardDeleteConfigActionPayload,
 }));
 
+export const dashboardCardConfigResizeCardIntent = createAction(
+  DashboardConfigAction.CARD_RESIZE,
+  (idx: number, span: gridSpans) => ({
+    payload: {
+      idx,
+      span,
+    } as DashboardResizeConfigActionPayload,
+  })
+);
+
 export interface CardConfig {
   name: string;
+  span: gridSpans;
```

Comment:
Since the shape of this state part changes, we should increment `_version` for this slicer?